lib/api/compiler/class-beans-compiler.php

```diff
@@ -547,28 +547,24 @@ public function replace_css_url_callback( $matches ) {
 		$base = $this->current_fragment;
 
 		// Separate the placeholders and path.
-		$explode_path = explode( '../', $matches[1] );
+		$paths = explode( '../', $matches[1] );
 
 		/**
 		 * Walk backwards through each of the the fragment's directories, one-by-one. The `foreach` loop
 		 * provides us with a performant way to walk the fragment back to its base path based upon the
 		 * number of placeholders.
 		 */
-		foreach ( $explode_path as $value ) {
+		foreach ( $paths as $path ) {
 			$base = dirname( $base );
 		}
 
-		// Rebuild path.
-		$replace      = preg_replace( '#^\/#', '', $explode_path );
-		$rebuilt_path = end( $replace );
-
 		// Make sure it is a valid base.
 		if ( '.' === $base ) {
 			$base = '';
 		}
 
 		// Rebuild url and make sure it is a valid one using the beans_path_to_url function.
-		$url = beans_path_to_url( trailingslashit( $base ) . $rebuilt_path );
+		$url = beans_path_to_url( trailingslashit( $base ) . ltrim( end( $paths ), '/' ) );
```

Comment:
>Use untrailingslashit() instead of ltrim()
We're actually removing the leading `/` before concatenating it to the base.  But we could improve the `ltrim` here to remove `\` or `/`.